topydo/lib/Colors.py

```diff
@@ -28,7 +28,7 @@ def __init__(self):
         self.metadata_color = config().metadata_color()
         self.link_color = config().link_color()
 
-    def _int_to_ansi(self, p_int, p_decorator='normal'):
+    def _int_to_ansi(self, p_int, p_decorator='normal', p_safe=False):
```

Comment:
I think the default for `p_safe` should be `True`.

Please also update the docstring to explain the notion of safe.